src/jit/compiler.cpp

```diff
@@ -4713,165 +4713,165 @@ unsigned getMethodBodyChecksum(__in_z char *code, int size)
 }
 
 
-int           Compiler::compCompileHelper (CORINFO_MODULE_HANDLE            classPtr,
-                                           COMP_HANDLE                      compHnd,
-                                           CORINFO_METHOD_INFO            * methodInfo,
-                                           void *                         * methodCodePtr,
-                                           ULONG                          * methodCodeSize,
-                                           CORJIT_FLAGS                   * compileFlags,
-                                           CorInfoInstantiationVerification instVerInfo)
-    {
-        CORINFO_METHOD_HANDLE methodHnd = info.compMethodHnd;
+int           Compiler::compCompileHelper(CORINFO_MODULE_HANDLE            classPtr,
+                                          COMP_HANDLE                      compHnd,
+                                          CORINFO_METHOD_INFO            * methodInfo,
+                                          void *                         * methodCodePtr,
+                                          ULONG                          * methodCodeSize,
+                                          CORJIT_FLAGS                   * compileFlags,
+                                          CorInfoInstantiationVerification instVerInfo)
+{
+    CORINFO_METHOD_HANDLE methodHnd = info.compMethodHnd;
```

Comment:
I didn't touch this method but while creating PR i have noticed there were formatting changes.  That was a bit of surprise for me too.  It could be an accidental change or VS editor might have auto formatted it.

I disagree that one has to also bring a method to CC standards just because they edited the formatting (intentionally or unintentionally).